cluster/gce/gci/configure-helper.sh

```diff
@@ -239,6 +240,10 @@ function create-master-auth {
   if [[ -n "${KUBE_PROXY_TOKEN:-}" ]]; then
     replace_prefixed_line "${known_tokens_csv}" "${KUBE_PROXY_TOKEN},"              "system:kube-proxy,uid:kube_proxy"
   fi
+  if [[ -n "${KUBE_CSR_TOKEN:-}" ]]; then
+    replace_prefixed_line "${known_tokens_csv}" "${KUBE_CSR_TOKEN},"                "system:node-bootstrap-secret,uid:node-bootstrap-secret,system:node-bootstrapper"
```

Comment:
if this isn't a username or group we intend to standardize (and I don't think it is), then omit the system: prefix... it's GCE-specific, like the admin username above.